From 4b9785fdf39627d38d59e28919def2df7718e57f Mon Sep 17 00:00:00 2001
From: Tianshu Qiu <tian.shu.qiu@intel.com>
Date: Thu, 16 Apr 2015 16:06:43 +0800
Subject: [PATCH 359/360] atomisp2: Remove redundant code

This patch removes the redundant code when assigning
the parameter update flag for DVS 6axis config.
The update flag is already updated in the function
atomisp_cp_dvs_6axis_config(). No need to assign it
outside.

Change-Id: I5456b3e8d2244d014dbf03f2f7a1f39a75767d75
Fixes-Issue: #JIMINAN-31158
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/51555
Approver: Bin B Han <bin.b.han@intel.com>
Reviewed-by: Bin B Han <bin.b.han@intel.com>
Integrator: Wei Tao <wei.tao@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Build: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
Maintainer: Wei Tao <wei.tao@intel.com>
---
 .../drivers/media/pci/atomisp2/atomisp_driver/atomisp_ioctl.c       | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_ioctl.c b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_ioctl.c
index 07a116c..16351ea 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_ioctl.c
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_ioctl.c
@@ -2769,12 +2769,8 @@ static long atomisp_vidioc_default(struct file *file, void *fh,
 	case ATOMISP_IOC_S_DIS_VECTOR:
 		err = atomisp_cp_dvs_6axis_config(asd, arg,
 				&asd->params.css_param, true);
-		if (!err && arg) {
-			asd->params.css_param.update_flag.dvs_6axis_config =
-				(struct atomisp_dvs_6axis_config *)
-				asd->params.css_param.dvs_6axis;
+		if (!err && arg)
 			asd->params.css_update_params_needed = true;
-		}
 		break;
 
 	case ATOMISP_IOC_G_ISP_PARM:
-- 
1.9.1

